brew_view/base_handler.py

```diff
@@ -41,8 +55,22 @@ def set_default_headers(self):
             self.set_header("Access-Control-Allow-Headers", "Content-Type")
             self.set_header("Access-Control-Allow-Methods", "GET, POST, PATCH, DELETE, OPTIONS")
 
+    @property
+    def current_time_millis(self):
+        """Current time in Milliseconds."""
+        return int(round(time.time() * 1000))
+
+    @property
+    def prometheus_endpoint(self):
+        """Removes Mongo ID from endpoint."""
+        to_return = self.request.path
+        for mongo_id in re.findall(self.MONGO_ID_PATTERN, self.request.path):
+            to_return = to_return.replace(mongo_id, '<ID>')
+        return to_return
+
     def prepare(self):
         """Called before each verb handler"""
+        self.request.headers['created_time_ms'] = self.current_time_millis
```

Comment:
Does this need to be on a header? Looks like it just gets deleted later...

Reply:
Yeah, I did not know I can just add things to the `self.request`. I will do that tonight.